controllers/todos.js

```diff
@@ -44,7 +41,10 @@ exports.findAllTodos = async function(req,res){
 
 exports.findTasksFromTodoId = async function(req,res){
     try{
-        const searchData = req.body;
+        const searchData = {};
+        searchData.id = req.body.id?+req.body.id:"Error";
+        searchData.skip = req.query.skip?+req.query.skip:"Error";
```

Comment:
Totalmente opcional, mas vc não acha essa formatação ruim de ler?

Que tal?
```suggestion
        searchData.skip = req.query.skip ? +req.query.skip : "Error";
```

Eu não vi se vc usa um `linter` nesse projeto (uma tool que faz uma análise do seu código e sugere algumas boas práticas, ou formatações maneiras), mas vc pode dar olhada e colocar o [eslint](https://eslint.org/), se quiser.

Ele é o mais usado por ai, tem extensão pro VSCode e pá, bem maneiro :B

A galera costuma usar esses linters em projetos com mais de 1 desenvolver, não só pq ele aponta algumas boas práticas (que as vezes previne bugs), mas tb pq força todo mundo a usar o msm padrão de formatação (pq cada um tem um gosto pessoal de como prefere formatar o código) pra maximizar a cosistência do código, e pra evitar dev cricri de ficar gastante energia a toa em Code Review (ver Bike [Shedding](https://en.wikipedia.org/wiki/Law_of_triviality), que diz que a galera tende a gastar 80% da energia discutindo coisa trivial, não essencial).

Reply:
Pois é. Eu concordo contigo. Eu confesso que achei essa parte do codigo bem feia também. Mas não me veio à cabeça nenhuma outra alternativa: porque imaginei que tornar essa parte mais tiposa também não deixaria ele mais bonito. 
O que eu percebi nessa parte do código, foi que eu criei meio que um "adaptador"... Tenho minhas duvidas se era realmente necessário.
Coisas como "DTO" e objetos intermediarios ficaram faltando e dai as coisas ficaram um pouco 'a Deus dará'.
Inclusive, ma man, vejo que voce anota e recupera BASTANTE informação de coisas e conceitos úteis. A forma com que eu guardo atualmente o conteúdo é meio "foda" de achar. Como voce se organiza? Google drive? Algum app de nota?

Reply:
No outro commit vou pesquisar maneiras de melhorar.
(lembrando que o objetivo do exercicio é aprender e não só passar batido)